### PR TITLE
Route fs edit list notifications through service

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco 

cc @strib

### CODE REVIEW

File: go/libkb/keyring.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ func (k *KeyringFile) Load() error {
 		return err
 	}
 	if file != nil {
+		defer file.Close()
```
Comment (patrickxb) [inline go/libkb/keyring.go]: I found this unclosed file while working on this.

**Comment by maxtaco** [comment]
👍 Nice!!!

**Comment by patrickxb** [comment]
@keybase/react-hackers new notification type, please help fix flow: https://ci.keybase.io/job/client/job/PR-4126/1/execution/node/140/log/

**Comment by patrickxb** [comment]
https://ci.keybase.io/job/client/job/PR-4126/1/execution/node/140/log/

**Comment by strib** [comment]
Thanks!  Looks good to me mod CI.

**Pull request merged**